Self-Consistent-Field/vcd.py

```diff
@@ -0,0 +1,874 @@
+psi_au2amu = psi4.constants.au2amu                                                                              # m_e -> amu
+#
+# Dipole moment type conversions
+psi_dipmom_au2debye = psi4.constants.dipmom_au2debye                                                            # e a0 -> D (C m)
+conv_dip_grad_au2DAamu = (psi_dipmom_au2debye / psi_bohr2angstroms) * (1 / np.sqrt(psi_au2amu))                 # (e a0 / a0)/(m_e^(1/2)) -> (D/A)/(amu^(1/2))
+conv_cgs = 6.46047502185 * (10**(-36))                                                                          # (e a0)^2 -> (esu * cm)^2
+#
+# IR frequency conversions
+omega2nu = 1./(psi_c*2*np.pi)                                                                                   # omega (s^-1)  -> nu (cm^-1)
+psi4_hartree2wavenumbers = psi4.constants.hartree2wavenumbers                                                   # Eh -> cm^-1
+conv_ir_au2DAamu = conv_dip_grad_au2DAamu ** 2                                                                  # (e a0 / a0)^2/m_e -> (D/A)^2/amu
+conv_freq_au2wavenumbers = np.sqrt((1 / psi_au2amu) * hartree2joule / (sqbohr2sqmeter * amu2kg)) * omega2nu     # (Eh / (bohr^2 m_e)) -> cm^-1
+conv_freq_wavenumbers2hartree = 1 / psi4_hartree2wavenumbers                                                    # cm^-1 -> Eh
+
+# Convert IR intensities from (D/A)^2/amu to km/mol:
+#
+# Conversion factor for taking IR intensities from ((D/A)^2 / amu) to a.u. ((e a0 / a0)^2 / me):
+conv_kmmol = (1 / conv_dip_grad_au2DAamu) ** 2
+# Multiply by (1 / (4 pi eps_0)) in a.u. (1 / (e^2 / a0 Eh)) to get to units of (Eh a0 / me)
+conv_kmmol *= 1
+# Multiply by (Na pi / 3 c^2) in a.u. (Na = mol^-1; c = Eh / alpha^2 me) to get to units of a0/mol
+conv_kmmol *= psi_na * np.pi * (1 / 3) * psi_alpha**2
+# Convert to units of km/mol
+conv_kmmol *= psi_bohr2m * (1 / 1000)
+#
+conv_ir_DAamu2kmmol = conv_kmmol                                                                                # (D/A)^2/amu -> km/mol
+
+
+# Specify Molecule
+#mol = psi4.geometry("""
+#    O            0.000000000000     0.000000000000    -0.075791843897
+#    H            0.000000000000    -0.866811832375     0.601435781623
+#    H            0.000000000000     0.866811832375     0.601435781623
+#    symmetry c1
+#    units ang
+#    noreorient
+#""")
+#mol = psi4.geometry("""
+#    O     0.000000000000     0.000000000000    -0.134503695264
+#    H     0.000000000000    -1.684916670000     1.067335684736
+#    H     0.000000000000     1.684916670000     1.067335684736
+#    symmetry c1
+#    units bohr
+#    noreorient
+#""")
+#mol = psi4.geometry("""
+#    O           -0.0000000000        1.3127774000       -0.1092016150
+#    O           -0.0000000000       -1.3127774000       -0.1092016150
+#    H            1.4385007800        1.7079033100        0.8736129170
+#    H           -1.4385007800       -1.7079033100        0.8736129170
+#    symmetry c1
+#    units bohr
+#    noreorient
+#""")
```

Comment:
Any reason to leave these commented lines in?

Reply:
Nope. Just needed further cleaning up.